src/Shell/Task/ModelTask.php

```diff
@@ -273,9 +267,15 @@ public function findBelongsTo($model, array $associations)
                         $tmpModelName = Inflector::camelize($found);
                     }
                 }
+                $foreignKeys = (array)TableRegistry::get($tmpModelName)->primaryKey();
```

Comment:
There is an issue. If `$schema` contains `webservice_id` and there isn't a `webservices` table in the database, it will throw an exception. I corrected only a `parent_id` issue, as I overlooked the above `if` block.